src/implementations/drip-config.ts

```diff
@@ -0,0 +1,102 @@
+import { Address } from '@project-serum/anchor';
+import { PublicKey } from '@solana/web3.js';
+import { DripConfig, Token, Vault, VaultProtoConfig } from '../interfaces';
+import { ClientEnv, Network } from '../models';
+import { Configuration, DefaultApi } from '@dcaf-labs/drip-ts';
+
+function getApiUrl(network: Network, clientEnv: ClientEnv): string {
+  switch (network) {
+    case Network.Mainnet:
+      return 'https://api.drip.dcaf.so';
+    case Network.Devnet:
+      switch (clientEnv) {
+        case ClientEnv.Production:
+          return 'https://devnet.api.drip.dcaf.so';
+        case ClientEnv.Staging:
+          return 'https://drip-backend-devnet-staging.herokuapp.com';
+      }
+  }
+  return 'localhost';
```

Comment:
Let's inject this URL in somehow

Reply:
Theres two ways to initialize it, one using network and client env which will derive the url as shown here, but there is an alternative to initialize via a `url` directly. `new DripConfigImpl(url)`